test/files/run/t7852.scala

```diff
@@ -17,7 +17,7 @@ object Test extends BytecodeTest {
       assert(got == expected, s"$methodName: expected $expected but got $got comparisons")
     }
     test("string", expected = 0)
-    test("module", expected = 0)
+    test("module", expected = 2)
```

Comment:
it would be good to extend this test and
  - rename `module` to `predefAlias`?
  - add a version of the current `def module` where `Nil` is actually accessed as module (`scala.colleciton.immutable.Nil`)